node/subsystem-types/src/messages.rs

```diff
@@ -875,8 +875,77 @@ pub enum GossipSupportMessage {
 	NetworkBridgeUpdate(NetworkBridgeEvent<net_protocol::GossipSupportNetworkMessage>),
 }
 
+/// Request introduction of a candidate into the prospective parachains subsystem.
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct IntroduceCandidateRequest {
```

Comment:
I don't see this in the commit we are backporting:

https://github.com/paritytech/polkadot/pull/5999/commits/6fa7b802d406d217243a8d5009419d5b7df31b4f#diff-090f011f1ffd771e361f4592c9631d505c6b9329062fadc2a49f34791e7a7ccb

It also isn't being used anywhere else in this PR.